parsons/utilities/api_connector.py

```diff
@@ -96,9 +96,12 @@ def get_request(self, url, params=None):
 
         r = self.request(url, "GET", params=params)
         self.validate_response(r)
-        logger.debug(r.json())
 
-        return r.json()
```

Comment:
This is a breaking change (though mostly internal) - can we switch this to passing in a `format` parameter that defaults to 'json', but if not 'json' but instead 'content', gets `r.content`?